pennylane/wires.py

```diff
@@ -352,11 +363,15 @@ def shared_wires(list_of_wires):
                     "Expected a Wires object; got {} of type {}.".format(wires, type(wires))
                 )
 
+        first_wire = list_of_wires[0]
```

Comment:
just to be clear, perhaps change name to something like `first_wires` or `first_wires_obj`?

Reply:
changed.